meinberlin/config/settings/base.py

```diff
@@ -577,3 +577,8 @@
 CELERY_RESULT_BACKEND = "redis://localhost:6379"
 CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
 CELERY_RESULT_EXTENDED = True
+
+# BO external footer
+BERLIN_FOOTER_URL = (
+    "https://www.berlin.de/rbmskzl/aktuelles/__i9/std/landesfooter.inc?js=0"
+)
```

Comment:
@goapunk can I just use the url here?

Reply:
@philli-m why not? that's also how we do it in gd I think